.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
Do we want to commit these VS Code settings too?

Reply:
We're gonna have to set them manually each time, or I have to make another PR. I'd rather get them in sooner than later.